modules/nf-core/mageck/mle/main.nf

```diff
@@ -27,7 +26,6 @@ process MAGECK_MLE {
     mageck \\
         mle \\
         $args \\
-        --threads $task.cpus \\
```

Comment:
Why is this losing its multi-threading? In which case, does it really need `process_high`?